### ISSUE TITLE
colour contrast is not adequate and is a problem for colourblind

### ISSUE THREAD

**Issue by LpmRaven** [smashingmagazine/redesign#135]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
<img width="795" alt="screen shot 2017-03-20 at 05 52 15" src="https://cloud.githubusercontent.com/assets/6523593/24089746/d3f07882-0d31-11e7-9fc9-72dd63dee1fc.png">

Colour contrast is below the recommended 4.5:1 for part of the primary colour scheme. I have included an image above of the contrast check between the two colours. Red-black colour blindness (although a small percentage of colour blindness) should be considered in the redesign. Using red and a dark colour is never a good idea as part of the main content/navigation.

**Comment by vitalyfriedman** [comment]
Thanks @LpmRaven — we are on it, as the issue has been considered and reported. Thank you! :-)

**Comment by jeankaplansky** [comment]
Please consider re-opening this issue. I was on the site today and noticed that the nav still turns white text to black on red when you hover over an item:

![image](https://user-images.githubusercontent.com/4044039/33284426-0eac9214-d37d-11e7-9e43-b189e9933297.png)

White on red is OK. Hovering and turning to black is definitely not OK. Smashing Magazine of all resources, needs to make sure that a11y details like this are addressed.

Either that or make sure this links to Issue 186 as a duplicate.

**Comment by LpmRaven** [comment]
Totally agree with @jeankaplansky. Thought this would have been sorted before release.
